docs/buildrun.md

```diff
@@ -237,15 +237,15 @@ The above allows users to get an overview of the building mechanism state.
 
 ### Understanding the state of a BuildRun
 
-A `BuildRun` resource stores the relevant information regarding the state of the object under `Status.Conditions`.
+A `BuildRun` resource stores the relevant information regarding the state of the object under `status.conditions`.
 
 [Conditions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties) allow users to easily understand the resource state, without needing to understand resource-specific details.
 
 For the `BuildRun` we use a Condition of the type `Succeeded`, which is a well-known type for resources that run to completion.
 
-The `Status.Conditions` hosts different fields, like `Status`, `Reason` and `Message`. Users can expect this fields to be populated with relevant information.
+The `status.conditions` hosts different fields, like `Status`, `Reason` and `Message`. Users can expect this fields to be populated with relevant information.
```

Comment:
Assuming we lowercase everything, should we also lowercase Status, Reason and Message?

Reply:
I dont have strong opinions, I only wanted to inline the paths, as they were mixed(lower/upper case). But I can also lower-case the fields.